launcher/go.mod

```
@@ -1,27 +1,27 @@
 // TODO: before merging, change module to "go.opentelemetry.io/contrib/launcher"
 module github.com/honeycombio/opentelemetry-go-contrib/launcher
 
-go 1.17
+go 1.18
```

Looks like upstream has moved over to v1.18 but this branch is still on v1.17 so CI is failing. Might be easiest to leave this on v1.17 for now and we can merge upstream into this branch to upgrade.

```suggestion
go 1.17
```